controls/radwebcam/features/recording-video.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Recording Video
+page_title: Recording Video
+description: This article shows how to record a video using the RadWebCam control.
+slug: radwebcam-features-recording-video
+tags: startrecording,capture
+published: True
+position: 0
+---
+
+# Recording Video
+
+To start recording a video you can press the "Start recording"  button or call the __StartRecording__ method of RadWebCam control. 
+
+![](images/radwebcam-features-recording-video-0.png)
+
+To enable video recording set the __RecordingFilePath__ property of the control. This is the path to the video where the recording will be stored.
+
+#### __[XAML] Example 1: Set the recording file path in XAML__
+{{region radwebcam-features-recording-video-0}}
+	<telerik:RadWebCam RecordingFilePath="C:\\temp\\video.mp4"/>
+{{endregion}}
+
+#### __[C#] Example 2: Set the recording file path in code__
+{{region radwebcam-features-recording-video-1}}
+	string videoFileName = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.Desktop), "video.mp4");
+	radWebCam.RecordingFilePath = videoFileName
+	
+	radWebCam.StartRecording();
+{{endregion}}
```

Comment:
Can we specify here if the file needs to be created before hand, or if there isn't such a file it will be created?